src/app/components/fee-row/fee-row.tsx

```diff
@@ -111,7 +111,7 @@ export function FeeRow(props: FeeRowProps): JSX.Element {
       {meta.error && (
         <ErrorLabel data-testid={SendFormSelectors.InputCustomFeeFieldErrorLabel}>
           <Text lineHeight="18px" textStyle="caption">
-            {meta.error}
+            <span data-testid={SendFormSelectors.InputCustomFeeFieldError}> {meta.error} </span>
```

Comment:
Same here, can be moved to component.

Reply:
Its needed for my test to find the element, spans are more reliable than text for the automation and can be found without failing and causing the test to flake.

Reply:
I would like to know how we can fix this rather than introducing a new span tag in these places. I don't feel this is necessary.

Reply:
I don't understand either. Why are spans more reliable if the test looks for a data attribute?